static/app/views/eventsV2/table/index.tsx

```diff
@@ -37,7 +37,7 @@ type TableState = {
   error: null | string;
   isLoading: boolean;
   pageLinks: null | string;
-  prevView: null | EventView;
+  prevView: null | [EventView, Location];
```

Comment:
@narsaynorath it'll be neat if there is a test to ensure pagination works as advertised.